src/Servers/IIS/IIS/src/Core/IISHttpContext.cs

```diff
@@ -26,6 +26,8 @@
 
 namespace Microsoft.AspNetCore.Server.IIS.Core
 {
+    using BadHttpRequestException = Microsoft.AspNetCore.Http.BadHttpRequestException;
```

Comment:
Style: Usings go outside the namespace where possible.

Reply:
It's not possible here. Placing the using outside of the namespace still uses the Obsolete version of BadHttpRequestException.

Reply:
Oh, because the obsolete one is in the Microsoft.AspNetCore.Server.IIS.Core namespace?